src/dispatch/individual/service.py

```diff
@@ -69,27 +67,17 @@ def create(*, db_session, individual_contact_in: IndividualContactCreate) -> Ind
     project = project_service.get_by_name(
         db_session=db_session, name=individual_contact_in.project.name
     )
-    terms = [
-        term_service.get_or_create(db_session=db_session, term_in=t)
-        for t in individual_contact_in.terms
-    ]
-    incident_priorities = [
-        incident_priority_service.get_by_name(
-            db_session=db_session, project_id=project.id, name=n.name
-        )
-        for n in individual_contact_in.incident_priorities
-    ]
-    incident_types = [
-        incident_type_service.get_by_name(db_session=db_session, project_id=project.id, name=n.name)
-        for n in individual_contact_in.incident_types
+
+    filters = [
+        search_filter_service.get(db_session=db_session, search_filter_id=f.id)
+        for f in individual_contact_in.filters
```

Comment:
I find the code easier to read if instead of using a single letter `f` we use a word `filter`.

Reply:
`filter` is a python reserved word so I try not to use it.